android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -89,15 +88,12 @@ object WorkerBinder {
 
   @JvmStatic
   @VisibleForTesting
-  open fun bind(mappedLifecycle: Observable<WorkerEvent>, worker: Worker): WorkerUnbinder {
-    val unbindFlow = MutableSharedFlow<WorkerEvent>(0, 1, BufferOverflow.DROP_OLDEST)
-
-    val workerLifecycle = merge(mappedLifecycle.asFlow(), unbindFlow)
-      .transformWhile { emit(it) ; it != WorkerEvent.STOP }
-      .asObservable()
-
-    bindToWorkerLifecycle(workerLifecycle, worker)
-    return WorkerUnbinder { unbindFlow.tryEmit(WorkerEvent.STOP) }
+  public fun bind(mappedLifecycle: Observable<WorkerEvent>, worker: Worker): WorkerUnbinder {
+    val d = mappedLifecycle
```

Comment:
optional nit: Maybe we could update `d` with a more meaningful name

Reply:
Good suggestion. A few chars more is worth the readability improvement.